dbx/sync/snapshot.py

```diff
@@ -0,0 +1,64 @@
+from collections import namedtuple
+
+from watchdog.utils.dirsnapshot import DirectorySnapshot
+
+SnapshotDiff = namedtuple(
```

Comment:
why not a dataclass (https://docs.python.org/3/library/dataclasses.html), which is essentially the same thing, but more readable?

Reply:
sure will do, sometimes I forget this exists :)